solr/CHANGES.txt

```diff
@@ -140,6 +140,8 @@ Other Changes
 
 * SOLR-15861: ConcurrentUpdateSolrClient should work with ManagedExecutorService (Sammy Chu, Kevin Risden)
 
+* SOLR-16562: Upgrade to Caffeine 3.1.2 (Kevin Risden)
```

Comment:
Probably want to mention the other dependency upgrades as well.

Reply:
definitely!  Dependency changes need to be very visible in a JIRA issue & CHANGES.txt

Reply:
Made this 3 commits since the woodstox stuff was done via contributor PR. Reused the PR branch and filed the jiras to make it all clear what got upgraded in versions.props.

* SOLR-16579 - Jackson 2.14.1
* SOLR-16578 - errorprone 2.16